src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsPluginComponent.kt

```diff
@@ -68,6 +73,9 @@ class SettingsPluginComponent {
      */
     private fun stylizePanel() {
 
+        // Add description to build Path
+        buildPathTextField.toolTipText = "Directs EvoSuite to the compiled classes. Default: target/classes"
```

Comment:
I know that this is how you did in the other settings. However, I  am thinking if tooltip should also appear when you hover over the text "Select the compilation  path:" and not just the text field. But it is a minor detail. So yeah.

Reply:
This is definitely food for thought. Though, I personally think the user is more likely to hover over the setting itself.

Reply:
Yeah, I usually hover over the name because I didn't know that the text fields can also have tooltips. But we can fix it later.